deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+        ```
+        ALERT_HOSTNAME=<NEW_HOST_NAME>
+        ```
+    - Editing overrides file:
+        ```
+            environment:
+                - ALERT_HOSTNAME=<NEW_HOST_NAME>
+        ```
+- Do not add the protocol a.k.a scheme to the value of the variable.
+    - Good: ```ALERT_HOSTNAME=myhost.example.com```
+    - Bad: ```ALERT_HOSTNAME=https://myhost.example.com```
+
+### Alert Logging Level Variable
+To change the logging level of alert add the following environment variable to your deployment. 
+
+- Editing environment file: 
+    ```ALERT_LOGGING_LEVEL=DEBUG```
+- Editing overrides file: 
+    ```
+        environment: 
+           - ALERT_LOGGING_LEVEL=DEBUG
+    ```
+
+- Set the value to one of the following: 
+    - DEBUG
+    - ERROR
+    - INFO
+    - TRACE
+    - WARN
+
+### Email Channel Environment Variables
+A majority of the Email Channel environment variables that can be set are related to JavaMail configuration properties.  You can find the JavaMail properties here: 
+
+[JavaMail Properties](https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html)
+
+- The Email Channel environment variables have a prefix of ```ALERT_CHANNEL_EMAIL_```
+- The remaining portion of the variable, after the prefix, map to the JavaMail properties if you replace '_' with '.'
+
+Examples:
+- ALERT_CHANNEL_EMAIL_MAIL_SMTP_HOST maps to 'mail.smtp.host'
+- ALERT_CHANNEL_EMAIL_MAIL_SMTP_PORT maps to 'mail.smtp.port'
+
+### Environment Variable Classifications
+There are certain classifications with the environment variables. The variables have a specific naming convention:
+```ALERT_<CLASSIFICATION>_<ITEM_NAME>_<CONFIGURATION_PROPERTY>```
+- Provider:  The environment variables to configure these components start with ALERT_PROVIDER_
+- Channel: The environment variables to configure these components start with ALERT_CHANNEL_
+- Component: The environment variables to configure these components start with ALERT_COMPONENT_
+
+Examples:
+These are some examples of what can be set. the blackduck-alert.env file has a more comprehensive list.
+- Provider: 
+    - ALERT_PROVIDER_BLACKDUCK_BLACKDUCK_URL= The URL for the Black Duck server.
+- Channel:
+    - ALERT_CHANNEL_JIRA_CLOUD_JIRA_CLOUD_URL= The URL for the Jira Cloud server.
+    - ALERT_CHANNEL_EMAIL_MAIL_SMTP_HOST= The SMTP host used to send email messages.
+- Component: 
+    - ALERT_COMPONENT_SETTINGS_SETTINGS_LDAP_ENABLED= Boolean to determine if LDAP authentication is used.
+
+## Advanced Configuration
+This section describes some advanced configuration settings for the Alert server.
+
+### Changing Server Port
+If Alert should not be running on it's default port of 8443, then this section describes what you have to change in order to use a different port.
+
+For this advanced setting since there are more than just environment variables that need to be set this should be performed by editing the ```docker-compose.local-overrides.yml``` file.
```

Comment:
probable more clear if you add commas here and change the end "For this advanced setting, since there are more than just environment variables that need to be set, edit the ```docker-compose.local-overrides.yml``` file."